source/help/messaging/sending-messages.rst

```diff
@@ -13,7 +13,7 @@ Write a message by typing into the text input box, then press ENTER to send it.
 Replying to Messages
 ---------------------
 
-Click the **[...]** link next to a message, then click **Edit**. This keeps conversations organized in threads.
+Click the **[...]** link next to a message, then click **Reply**. This keeps conversations organized in threads.
```

Comment:
It's faster and easier for users to simply click the reply icon 
![image](https://user-images.githubusercontent.com/20325173/28473447-6d448abe-6df9-11e7-8ca7-9ef93f397ec1.png)

Also, you should introduce the right-hand sidebar, by saying something about what happens when you reply. The screen layout and terminology isn't covered anywhere in the end-user docs so "right-hand sidebar" should probably have a bit of an explanation.

"To adjust the width ..." implies that you can use the mouse to drag to a new width instead of what really happens. Maybe something like "To expand the right-hand sidebar to its full width..."

And an image of the expand/shrink icon would be helpful too. 
![image](https://user-images.githubusercontent.com/20325173/28473616-3aa4a714-6dfa-11e7-8ce3-f3c874c5d48a.png)